src/main/java/com/novoda/notils/logger/simple/Log.java

```diff
@@ -177,8 +177,8 @@ private static String getDetailedLog(String msg, int depth) {
         Thread current = Thread.currentThread();
         final StackTraceElement trace = current.getStackTrace()[depth];
         final String filename = trace.getFileName();
-        return "[" + current.getName() + "][" + filename.substring(0, filename.length() - DOT_CLASS) + "."
-                + trace.getMethodName() + ":" + trace.getLineNumber() + "] " + msg;
+        return "[" + current.getName() + "][" + "(" + filename.substring(0, filename.length() - DOT_CLASS) + ".java:"
+                + trace.getLineNumber() + ")." + trace.getMethodName() + "] " + msg;
```

Comment:
why move the line number and method name around? or is this involved in the linking